### DISCUSSION TITLE
Launchpad v12 Liftoff Activity

### DISCUSSION THREAD

**Discussion by walkerford** [protocol/launchpad#531] [General]
Welcome Launchpad v12 residents!  Please answer any of the follow questions in this discussion:

- What do you like most about web3?
- What do you find most confusing about web3?
- What are you most excited to learn about in web3?

**Comment by ErickWa** [comment]
* The coolest thing about Web3 is that decentralization enables true agency at the user / individual level, rather than the organization / corporation level.
* I find it confusing in terms of how it's (dis)organized
* I'm excited to learn about patterns that work (and don't work)

**Comment by web3risk** [comment]
I'm interested in how web3 decentralized trust will solve security and risk problems traditionally solved with web2.

**Comment by ajcorrado** [comment]
I love the idea of individual ownership of and agency over their data. I'm most excited to learn the novel and creative use cases other projects in the launchpad program are developing for.

**Comment by dhuseby** [comment]
I like that applying cryptography is where we start when we build with web3 technologies. I think some web3 applications get too meta and loose sight of being decentralized. I'm excited to solve a few of the remaining unsolved nine fundamental problems of decentralization so that we can create magically useful tools that only require an internet connection without any concern for servers, accounts, etc.

**Comment by travis** [comment]
* my favorite thing about web3 is the wide variety of fundamentally new primitives it gives us to build applications 
* this is also the most confusing thing :-p 
* I'm excited to continue learning how to tame this wild toolshed of new exotic cryptographic and computational devices and bring them into service of all of the web's users

**Comment by mzkrasner** [comment]
- What I appreciate most about Web3 are the coordination possibilities permissionless, trustless systems unlock
- What I find confusing (yet also exciting) are the incentive mechanisms or gamified aspects that lie beneath permissionless systems, and how incentives conflict or align with actual human behavior under various conditions

**Comment by fridrik01** [comment]
- I am excited for the new bleeding edge technologies that enable decentralised solutions as competitive alternatives to the big enterprises/corps
- Complex ecosystem and tooling
- Fundamental libraries with specific deep dives in topics that are relevant to my role and interests

**Comment by nelsonmwania** [comment]
What I like most about web3 is the tech eliminates intermediaries (banks, big tech, etc), and promotes transparency, openness and access around the world.

**Comment by JeanineBoselli** [comment]
What I value most about web3 is the autonomy, openness, and how it promotes direct ownership.
